.github/workflows/ci.yaml

```
@@ -0,0 +1,81 @@
+      - name: Login to ghcr.io
+        uses: docker/login-action@v3.5.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+      - name: Build image
+        run: |
+          make hook-build-image
+      - name: Release branch image
+        if: github.event_name == 'pull_request' && github.actor != 'dependabot[bot]'
+        run: |
```

actionlint (1.7.9)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)